# Data for each sherd processed by robot arm
std_msgs/Header header
sensor_msgs/Image archival_photo
float32 mass
bool incomplete
